modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -54,12 +62,15 @@
    *   The event dispatcher.
    * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
    *   The route match.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, EventDispatcherInterface $event_dispatcher, RouteMatchInterface $route_match) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EventDispatcherInterface $event_dispatcher, RouteMatchInterface $route_match, EntityTypeManagerInterface $entity_type_manager) {
```

Comment:
Let's inject the entity type manager before the event dispatcher, it's more essential (and it's alphabetic),